### PR TITLE
fix: [CM-683][Sale Widget] Fallback token images

### PR DESCRIPTION
**Pull request by jhesgodi**
# Summary
Apply fallback image component to all token images across checkout widgets

### CODE REVIEW

**Review by dreamoftrees — APPROVED** [review]
Nice work @jhesgodi 👍

File: package.json [hunk 1]
```diff
@@ -16,6 +16,7 @@
     "eslint": "^8.40.0",
     "eslint-config-airbnb": "^19.0.4",
     "eslint-config-airbnb-typescript": "^17.0.0",
+    "eslint-plugin-react-refresh": "latest",
```
Comment (jhesgodi) [inline package.json:19]: added missing package, picked up from merging `main` was causing eslint to fail

**Review by jwhardwick — APPROVED** [review]
Nice abstraction 💯

**Review by dreamoftrees — APPROVED** [review]
Great work @jhesgodi

**Pull request merged**